dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -0,0 +1,29 @@
+resource "kafka_topic" "uswitch.data.v1" {
+  name = "customer_proposition.uswitch.data.v1"
+
+  replication_factor = 3
+  partitions         = 15
+
+  # infinte retention
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
+    "local.retention.ms"    = "3600000"
```

Comment:
Can you please add comments on what these values are in a more human friendly unit (days, MB)